src/app.js

```diff
@@ -1 +1,67 @@
+/* eslint-disable no-console */
 'use strict';
+
+const readline = require('readline');
+
+const input = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+let bulls = 0;
+let cows = 0;
+
+const getRandomNumber = () => {
+  let result = '';
+
+  while (result.length < 4) {
+    const randomNumber = Math.floor(Math.random() * 10);
+
+    if (!result.includes(randomNumber)) {
+      result += randomNumber;
+    }
+  }
+
+  return result;
+};
+
+const randomNum = getRandomNumber();
+
+console.log(randomNum);
+
+const getBullsAndCows = (guess) => {
+  const guessArr = guess.split('');
+  const guessIsCorrect = guessArr.every((num, ind) => (
```

Comment:
from the function name now it seems to me that it makes the guess (cause it has a verb at the beginning). After changing places of two words - it is now clear that we're dealing with a boolean variable
```suggestion
  const isGuessCorrect = guessArr.every((num, ind) => (
```